examples/webgl/raycaster/index.html

```diff
@@ -52,7 +52,7 @@
   event.preventDefault();
   mouse.x = (event.clientX / window.innerWidth) * 2 - 1;
   mouse.y = -(event.clientY / window.innerHeight) * 2 + 1;
-js.document.addEventListener('mousemove', pyodide.create_proxy(onMouseMove))
+js.document.addEventListener('mousemove', pyodide.ffi.create_proxy(onMouseMove))
```

Comment:
this change looks unrelated to this PR? What is it about?

Reply:
Yeah, this isn't related but basically functions like `create_proxy ` and `to_js` are now a part of `pyodide.ffi` and are not to be imported directly from `pyodide`. We can but there is a depreciation warning otherwise. Since it was a small change, I fixed it here since a dedicated PR to change it at all places has already been merged, but this place was missed apparently.

Reply:
Resolving this convo, but thanks for asking!